javascript/introduction/introduction.md

```diff
@@ -1,23 +1,23 @@
 
 You've already completed the [Foundations course](/courses/foundations/#section-the-front-end), right? Good, because now we'll be moving at warp speed into new frontiers. This section will cover a lot of ground and your brain may meltdown a few times, but don't worry, that's just a flesh wound. Patch 'er up and keep going! When in doubt, build something.
 
 ### The Path
 
-So how is this course set up? It starts with a deeper look at the basics, just like the [Ruby Programming course](/courses/ruby-programming) did with Ruby. You don't need to have completed the Ruby Programming course or [the Ruby on Rails course](/courses/ruby-on-rails) to understand these basics but we won't be focusing deeply on the really basic coding items so it'll move quickly. You should, however, already have completed the [Foundations course](/courses/foundations), specifically the [Front End Section](/courses/foundations#the-front-end) before starting this course. You might also find this [Git Section](https://www.theodinproject.com/courses/ruby-programming#git) on more intermediate-level uses of Git helpful.
+How is this course set up? It starts with a deeper look at the basics, just like the [Ruby Programming course](/courses/ruby-programming) did with Ruby. You don't need to have completed the Ruby Programming course or [the Ruby on Rails course](/courses/ruby-on-rails) to understand these basics. We won't be focusing deeply on the really basic coding items, so it will move quickly. You should, however, already have completed the [Foundations course](/courses/foundations), specifically the [Front End Section](/courses/foundations#the-front-end) before starting this course. You might also find this [Git Section](https://www.theodinproject.com/courses/ruby-programming#git) on more intermediate-level uses of Git helpful.
 
 The last thing you'll do is a final project which integrates everything you've learned in all the courses of this curriculum. This is the kind of project that you'll be telling employers all about and a great chance to prove that you're now, officially, a serious web developer.  
 
 ### Format
 
-There's a lot to cover, but it's been broken up into bite-sized lessons and their accompanying projects. The projects will give you a chance to apply what you've learned and to show what you're capable of. After a few of them, you'll really start getting the hang of things.  
+There is a lot to cover, but this course has been course been broken up into bite-sized lessons and their accompanying projects. These projects will give you a chance to apply what you have learned and to show what you are capable of. After a few of them, you'll really start getting the hang of things.  
```

Comment:
So close.. "this course has been course been"

Reply:
Man I am having trouble with git hub. All the colors.